.dockerignore

```diff
@@ -0,0 +1,31 @@
+# Vagrant 
+Vagrantfile
+.vagrant
+
+#Test files
+**/tests*
+**/vcr_cassettes*
+**/pytest.ini
+.env.vcr.*
+.env.test
+
+# pyenv
+.python-version
+
+.vscode
+
+# Logs
+**/geckodriver.log
+**/log.txt
+
+#docker
+Dockerfile
+.dockerignore
+
+#git
+.git
+.gitignore
+
+
```

Comment:
You should add .venv to avoid copying your local virtual environment into the image

Reply:
This is now added